public/scripts/editor/js/bramble-editor.js

```diff
@@ -4,10 +4,11 @@ define(function(require) {
       ProjectRenameUtility = require("fc/project-rename"),
       ProjectFiles = require("fc/load-project-files"),
       FileSystemSync = require("fc/filesystem-sync"),
-      SyncState = require("fc/sync-state");
+      SyncState = require("fc/sync-state"),
+      SSOOverride = require("sso-override");
```

Comment:
I'm not sure I like this coupling...the editor should not have anything to do with authentication.
What I suggest is creating a separate module, something like `context.js'. That way, you can store references to for e.g. the project, the anon id, etc. and update/access them by reference